.github/workflows/automerge-pyup.yml

```diff
@@ -0,0 +1,37 @@
+name: "pyup Automerge"
+
+on:
+  pull_request:
+
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    
+    if: github.actor == 'pyup-bot'
+    steps:
+      - name: 'Wait for status checks'
+        id: waitforstatuschecks
+        uses: "WyriHaximus/github-action-wait-for-status@master"
+        with:
+          ignoreActions: automerge,WIP
```

Comment:
Ah sorry, WIP is a GitHub marketplace app that allows you to write [WIP] in the title to prevent merging. I had it in my test repo, and it blocked everything even on success, so I added it here in case you added WIP to cookiecutter-django. I can remove it if you'd like.